src/barcode_correct_sciatac.py

```diff
@@ -16,6 +16,84 @@
 import barcode_to_well
 import barcode_constants as bc
 
+#
+# Notes:
+#   o  to add new index sequence 'recipes', do the
+#      following
+#        o  add a function that finds the index sequences.
+#           There are existing functions with names such
+#           as header_parser_01(). See below.
+#        o  modify the function choose_header_parser() to
+#           select the new header_parser_<n> function
+#             o  add the new function to header_parser_dict
+#             o  increase the range in
+#                  if(index_map < 1 or index_map > 5)
+#        o  use --index_recipe command line parameter to
+#           run barcode_correct_sciatac.py using the new
+#           recipe.
+#
+
+def load_std_index_lists(args):
+    if args.two_level_indexed_tn5:
+        tagi7 = bc.nex_i7_two_level_indexed_tn5_list
+        pcri7 = bc.pcr_i7_two_level_indexed_tn5_list
+        pcri5 = bc.pcr_i5_two_level_indexed_tn5_list
+        tagi5 = bc.nex_i5_two_level_indexed_tn5_list
+    else:
+        if args.wells_384:
+            tagi7 = bc.lig_i7_list_384
+            pcri7 = bc.pcr_i7_list_384
+            pcri5 = bc.pcr_i5_list_384
+            tagi5 = bc.lig_i5_list_384
+        else:
+            tagi7 = bc.lig_i7_list
+            pcri7 = bc.pcr_i7_list
+            pcri5 = bc.pcr_i5_list
+            tagi5 = bc.lig_i5_list
+
+    return tagi7, pcri7, pcri5, tagi5
+
+
+def load_std_index_to_well_dicts(args):
+    if args.wells_384:
+        lig_i7_to_well = bc.lig_i7_to_well_384
+        lig_i5_to_well = bc.lig_i5_to_well_384
+        pcr_to_well = bc.pcr_to_well_384
+    else:
+        lig_i7_to_well = bc.lig_i7_to_well
+        lig_i5_to_well = bc.lig_i5_to_well
+        pcr_to_well = bc.pcr_to_well
+
+    return lig_i7_to_well, lig_i5_to_well, pcr_to_well
+
+
+def load_std_index_to_well_indexed_tn5_dicts(args):
+    if(args.two_level_indexed_tn5):
+        nex_two_level_indexed_tn5_to_well = bc.nex_two_level_indexed_tn5_to_well
+        pcr_two_level_indexed_tn5_to_well = bc.pcr_two_level_indexed_tn5_to_well
+    else:
+        nex_two_level_indexed_tn5_to_well = None
+        pcr_two_level_indexed_tn5_to_well = None
+    return nex_two_level_indexed_tn5_to_well, pcr_two_level_indexed_tn5_to_well
+
+
+def set_index_flags(args):
+    if args.two_level_indexed_tn5:
+        for i in range( 12, 384 ):
```

Comment:
Is there a reason it starts at 12? (I'm not very familiar with this pipeline/assay.)